index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="styles.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link
+    href="https://cdn.jsdelivr.net/npm/remixicon@4.2.0/fonts/remixicon.css"
+    rel="stylesheet"
+/>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+	<title>Personal Portfolio</title>
+</head>
+<body>
+  <header>
+		<nav id="navbar">
+		<ul class="nav-links">
+			<li><a href="#welcome-section" class="nav-link">Welcome</a></li>
+			<li><a href="#about" class="nav-link">About</a></li>
+			<li><a href="#projects" class="nav-link">projects</a></li>
+			<li><a href="#contact" class="nav-link">Contact</a></li>
+		</ul>
+     </nav>
+	</header>
+	<section id="welcome-section">
+		<h1>Hey, I am Hamasa Noorzai</h1>
+		<h2 class="note">A Web developer</h2>
+		<p class="info">
+			I am Hamasa, a junior developer and BCS student at the American University of Afghanistan. Passionate about technology, I excel in programming, data structures, and software engineering. I apply my skills html and css to real-world projects.
+		</p>
+		<button class="hire-me" type="button">Hire me</button>
+		<button class="lets-talk" type="button">Let's talk</button>
```

Comment:
- These two buttons do not have any functionality, remove them or add functionality to them
![Screenshot 2024-06-04 115509](https://github.com/Hamas2000/Personal-Portfolio/assets/96838030/bc90698f-98a3-43ef-876a-117957846601)